detox/index.d.ts

```diff
@@ -274,6 +274,55 @@ declare global {
             reuse?: boolean;
         }
 
+        /**
+         * A construct allowing for the querying and modification of user arguments passed to an app upon launch by Detox.
+         *
+         * @see AppLaunchArgs#modify
+         * @see AppLaunchArgs#reset
+         * @see AppLaunchArgs#get
+         */
+        interface AppLaunchArgs {
+            /**
+             * Modify the launch-arguments via a modifier object, according to the following logic:
+             *  - Concrete modifier properties would either set anew or override the value of existing properties with the same name, with
+             *    the specific value.
+             *  - Modifier properties set to either `undefined` or `null` would have the equivalent property cleared.
+             *
+             * @param modifier The modifier object.
+             *
+             * @example
+             * // With current launch arguments set to:
+             * // {
+             * //   mockServerPort: 1234,
+             * //   mockServerCredentials: 'user@test.com:12345678',
+             * // }
+             * device.modify({
+             *   mockServerPort: 4321,
+             *   mockServerCredentials: null,
+             *   mockServerToken: 'abcdef',
+             * };
+             * await device.launchApp();
+             * // => launch-arguments become:
+             * // {
+             * //   mockServerPort: 4321,
+             * //   mockServerToken: 'abcdef',
+             * // }
+             */
+            modify(modifier: object): void;
```

Comment:
Maybe `assign`, to be in line with `Object.assign` ?

Reply:
Well my line of though is that since for the undefined/null case we effectively delete properties, assigning isn't 100% descriptive. We're effectively modifying, perhaps even editing 🤔

Reply:
Okay.